packages/blade/src/components/Checkbox/Checkbox.tsx

```diff
@@ -213,6 +214,7 @@ const _Checkbox: React.ForwardRefRenderFunction<BladeElementRef, CheckboxProps>
         <BaseBox display="flex" flexDirection="column">
           <BaseBox display="flex" alignItems="center" flexDirection="row">
             <SelectorInput
+              hoverStyles={checkboxHoverVariants}
```

Comment:
I see you've already thought of using hoverStlyes. Nice 😄 . Lets use that everywhere then?
```suggestion
              hoverStyles={checkboxHoverStyles}
```

Reply:
Looking at your previous comment, i think token makes more sense. 

hoverStyles is a bit misleading becomes that object doesn't contain any CSS styles or rules instead just blade tokens. 

Let's do <Component hoverTokens={checkboxHoverTokens} \/>